src/controller/index.js

```diff
@@ -131,21 +130,55 @@ const drawQuestionMarks = (coordinates) => {
             }
         }
         else{
-            console.log(key, coordinates)
             drawNewText(key, coordinates[key])
         }
         
     }
 }
 
-const drawNewText = (txt, currCoordinates) => {
-    requestAnimationFrame(() => {
-        const ctx = imgCanvas.getContext('2d')
-        ctx.font = "15px Arial"
-        let coordinates = currCoordinates.split(' ')
-        ctx.fillText(txt, coordinates[0], coordinates[1]);
-    })   
+// const drawNewText = (txt, currCoordinates) => {
+//     requestAnimationFrame(() => {
+//         const ctx = imgCanvas.getContext('2d')
+//         ctx.font = "15px Arial"
+//         let coordinates = currCoordinates.split(' ')
+//         ctx.fillText(txt, coordinates[0], coordinates[1]);
+//     })   
     
+// }
+const drawTextBackground = async (ctx, txt, x, y, font, padding) => {
```

Comment:
duplicate code should probably be refactored, but it's fine for now since megan needs it soon